minecraft/src/main/java/org/quiltmc/loader/impl/game/minecraft/MinecraftGameProvider.java

```diff
@@ -474,6 +474,8 @@ public void launch(ClassLoader loader) {
 			targetClass = "org.quiltmc.loader.impl.game.minecraft.applet.AppletMain";
 		}
 
+		System.out.println(targetClass);
```

Comment:
Can you remove this, or change it to a more informative message, such as:
```suggestion
		Log.log(LogCategory.GAME_PROVIDER, "Launching using target class '" + targetClass + "'");
```

Reply:
Right, I put that there for debug stuff, I'll remove it